os/include/sys/socket.h

```diff
@@ -89,6 +85,24 @@ extern "C" {
 #define EXTERN extern
 #endif
 
+/****************************************************************************
```

Comment:
Isn't this already defined now at sockets.h? Please ignore if not the case. Thanks.